src/lib.rs

```diff
@@ -209,7 +209,7 @@ impl App {
             Event::Query(event::Query::Metrics) => match self.redis.connect().await {
                 Ok(mut redis) => {
                     if let Err(e) = redis
-                        .set(
+                        .lpush(
```

Comment:
What is the reasoning for switching to a list here?

Reply:
Because then it is possible to use blocking fetch `[brpop](https://redis.io/commands/brpop/)` instead of waiting specified time.

Reply:
Ok, one problem is that if the list doesn't get pop'd for any reason you end up growing the list forever.

Adding an `trim` after the push here should resolve that issue